update build-in-alert doc (#12644)
pingcap/docs: tidb-cloud/tidb-cloud-dm-precheck-and-troubleshooting.md

The following are alerts about Data Migration: 

- "Data migration job met error during data export"

>     - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.

Reviewer: ```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

- "Data migration job met error during data import"

>     - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.

Reviewer: ```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

- "Data migration job met error during incremental data migration"

>     - Recommended Action : Check the error message from data migration page, and refer to [Migration errors and solutions](#migration-errors-and-solutions) for help.

Reviewer: ```suggestion

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

```

- "Data migration job has been paused for more than 6 hours during incremental migration" 

>     - Recommended Action : If this alert occurs, you can resume the data migration job or ignore this alert directly.     

Reviewer: ```suggestion

    Recommended action: resume the data migration job or ignore this alert.     

```

- "Replication lag is larger than 10 minutes and stilling increasing for more than 20 minutes"

>     - Recommended Action : Contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for consultation.

Reviewer: ```suggestion

    - Recommended action: contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for help.

```

The following are alerts about Data Migration: 

- "Data migration job met error during data export"

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

- "Data migration job met error during data import"

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

- "Data migration job met error during incremental data migration"

    Recommended action: check the error message on the data migration page, and see [Migration errors and solutions](#migration-errors-and-solutions) for help.

- "Data migration job has been paused for more than 6 hours during incremental migration" 

    Recommended action: resume the data migration job or ignore this alert.     

- "Replication lag is larger than 10 minutes and stilling increasing for more than 20 minutes"

    - Recommended action: contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for help.

> 

Reviewer: ```suggestion
```
